README.md

```diff
@@ -224,6 +224,8 @@ AND r.history_index = 0;
 
 **Retrieve the UAST for all files from [HEAD](https://git-scm.com/book/en/v2/Git-Internals-Git-References#ref_the_ref)**:
 
+**Note:** this query is meant for the `web sql` interface. If you query `UAST()` from the CLI the result will be encoded binary data.
```

Comment:
I think this is a valuable notice but is a bit hidden between query examples.
What if we move it to the place where we introduced UAST?
:point_right: I'd say [`README.md::L135`](https://github.com/src-d/engine/pull/209/files#diff-04c6e90faac2675aa89e2176d2eec7d8R135)

Reply:
Makes sense, done in bf36d90